scripts/signoff-git-hook.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 set -euo pipefail
 
-# Automatically signs off your commits.
+# Used by vdev/src/commands/install_git_hooks.rs to
+# automatically sign off your commits.
```

Comment:
```suggestion
# Automatically sign off your commits.
```

The original text is still correct, technically. The `vdev` function just copies this script which is run by `git`.